autoreject/autoreject.py

```diff
@@ -294,10 +294,13 @@ def _compute_thresh(this_data, thresh_range, method='bayesian_optimization',
         from skopt import gp_minimize
         from sklearn.cross_validation import cross_val_score
 
-        def objective(thresh):
+        threshes = np.ptp(this_data, axis=1)
+
+        def objective(n_epochs):
+            thresh = np.sort(threshes)[n_epochs]
```

Comment:
maybe this is making it slower, let me move the `sort` outside the `objective` ...